python/kvikio/pyproject.toml

```diff
@@ -8,10 +8,10 @@ requires = [
     "cython>=3.0.0",
     "ninja",
     "scikit-build-core[pyproject]>=0.7.0",
-] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../dependencies.yaml and run `rapids-dependency-file-generator`.
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
 
 [project]
-name = "kvikio"
+name = "kvikio-cu11"
```

Comment:
is this OK to be committed?

Reply:
Nope you don't want to commit that either. The suffixes are generated by the scripts at build time (and in the future by [RBB](https://github.com/rapidsai/rapids-build-backend), still at build time).